src/sentry/scim/endpoints/utils.py

```diff
@@ -108,13 +108,15 @@ def validate_filter(self, filter):
 
 
 class OrganizationSCIMPermission(OrganizationPermission):
-    def has_object_permission(self, request: Request, view, organization):
-        result = super().has_object_permission(request, view, organization)
+    def has_object_permission(self, request: Request, view, organization_context):
+        result = super().has_object_permission(request, view, organization_context)
```

Comment:
Should we just passing in the organization instead of the context wrapper? Getting context sometimes and organization in other scenarios could be confusing for future us.

Reply:
Removed!